### ISSUE TITLE
Sync layouts for client

### ISSUE THREAD

**Issue by willdoran** [ushahidi/platform#884]
**Home: list**
- **Missing features:**

- [ ] "Mark as" button
- [ ] "Export" button
- [ ] "Share" button
- [ ] Icon beside role, Custom Roles UX should allow setting of an icon for the role
- [ ] Comments
- [ ] “via SMS” on Post item - data source origin identified - twitter, sms, etc. We don't currently capture or display this information and we probably should.
- [ ] Post type - colour type - allow user to set post type colour
	
- **CSS Issues:**
- [x] Edit button colour on firefox
- [x] Edit/collection button spacing
- [ ] Arrow beside “Who can see this”

**Home: Timeline**
- **CSS Issues:**
- [ ] Background colour for graph should be white but is transparent

**Add Post: Select Post Type**
-**CSS Issues:**
- [ ] Space between ‘edit post type’ and ‘select post type’

**Post Message:**
- **Missing:**
- [ ] Appears to be missing?
		
**Settings Datasource:**
- **CSS Issues:**
- [x] Email: layout of form needs divider between settings sections
- [x] Step and field title should both use either label or legend
- [x] “You need to configure this data source before you can enable it.” should look like an alert - add wrapping div

**Settings views:**
- [ ] Missing completely?

**Post types:**
- **CSS Issues:**
- [x] Steppers not needed
- [x] Delete icon needs refinement
- [x] Add post type button alignment

**Edit post type:**
- **CSS Issues:**
- [x] Buttons between sections removed
- [x] Alignment of form sections

**Import:**
- **CSS Issues:**
- [x] Post type should default to first one option

**Organize data:**
- **CSS Issues:**
- [x] Column names not capitalized
- [x] Dropdown width is too great

**Configure post:**
- **CSS Issues:**
- [ ] Not implemented yet

**Users:**
- **CSS Issues:**
- [ ] Search filters is shown on pattern library rather than Role filtering
- [ ] Filter is always shown
- [ ] View does not resize properly

**Comment by willdoran** [comment]
@rjmackay @jshorland 

@middle8media  and I went through Layout A to check the client against the pattern library and we found the following issues(Seth if I missed anything let me know). Most are CSS mistakes on the client that need to be brought in live with the pattern library. 

In some cases, the pattern library represent features that do not exist on the client - we should consider if we want to mark these for later addition or if they should be removed from the pattern library.

I'll try to clean this up more to make it clearer what might be missing but it may be beneficial when looking at the below to take a look at the pattern library and the client at the same time.

http://github.ushahidi.org/platform-pattern-library/assets/html/5_layouts/

**Home: list**
	

- **Missing features:**

		-"Mark as" button
		-"Export" button
		-"Share" button
		-Icon beside role **Custom Roles UX should allow setting of an icon for the role**
		-Comments
		-“via SMS” on Post item - data source origin identified - twitter, sms, etc. We don't currently capture or display this information and we probably should.
		-Post type - colour type - **allow user to set post type colour**
	

- **CSS Issues:**

		-Edit button colour on firefox
		-Edit/collection button spacing
		-Arrow beside “Who can see this”

**Home: Timeline**
	

- **CSS Issues:**

		-Background colour for graph should be white but is transparent

**Add Post: Select Post Type**

- **CSS Issues:**

		-Space between ‘edit post type’ and ‘select post type’

**Post Message:**

- **Missing:**

              -Appears to be missing?
		
**Settings Datasource:**

- **CSS Issues:**

		-Email: layout of form needs divider between settings sections
		-Step and field title should both use either label or legend
		-“You need to configure this data source before you can enable it.” should look like an alert - add wrapping div

**Settings views:**

		-Missing completely?

**Post types:**

- **CSS Issues:**

		-Steppers not needed
		-Delete icon needs refinement
		-Add post type button alignment

**Edit post type:**

- **CSS Issues:**

		-Buttons between sections removed
		-Alignment of form sections

**Import:**

- **CSS Issues:**

		-Post type should default to first one option

**Organize data:**

- **CSS Issues:**

		-Column names not capitalized
		-Dropdown width is too great

**Configure post:**

- **CSS Issues:**

		-Not implemented yet

**Users:**

- **CSS Issues:**

		-Search filters is shown on pattern library rather than Role filtering
		-Filter is always shown
		-View does not resize properly

**Comment by middle8media** [comment]
I am currently working on these for the Feb 14 sprint deadline.

**Comment by jshorland** [comment]
Great. Thanks, will and seth for this (I’m sure tedious) much needed effort. 

I would like to propose that we hold off on making decisions on the pattern library contents until we start incorporating Mustang designs after the UX studies. 


Jess Shorland
Product manager
jess@ushahidi.com

**Comment by middle8media** [comment]
CSS is sorted for now, well after https://github.com/ushahidi/platform-client/pull/78 is merged.

**Comment by rjmackay** [comment]
@willdoran Could you summarize what needs to be done in the description for this ticket? ideally use the checklist formatting so we can check things off.

**Comment by willdoran** [comment]
**Home: list**
- **Missing features:**

- [ ] "Mark as" button
- [ ] "Export" button
- [ ] "Share" button
- [ ] Icon beside role, Custom Roles UX should allow setting of an icon for the role
- [ ] Comments
- [ ] “via SMS” on Post item - data source origin identified - twitter, sms, etc. We don't currently capture or display this information and we probably should.
- [ ] Post type - colour type - allow user to set post type colour
	
- **CSS Issues:**
- [ ] Edit button colour on firefox
- [ ] Edit/collection button spacing
- [ ] Arrow beside “Who can see this”

**Home: Timeline**
- **CSS Issues:**
- [ ] Background colour for graph should be white but is transparent

**Add Post: Select Post Type**
-**CSS Issues:**
- [ ] Space between ‘edit post type’ and ‘select post type’

**Post Message:**
- **Missing:**
- [ ] Appears to be missing?
		
**Settings Datasource:**
- **CSS Issues:**
- [ ] Email: layout of form needs divider between settings sections
- [ ] Step and field title should both use either label or legend
- [ ] “You need to configure this data source before you can enable it.” should look like an alert - add wrapping div

**Settings views:**
- [ ] Missing completely?

**Post types:**
- **CSS Issues:**
- [ ] Steppers not needed
- [ ] Delete icon needs refinement
- [ ] Add post type button alignment

**Edit post type:**
- **CSS Issues:**
- [ ] Buttons between sections removed
- [ ] Alignment of form sections

**Import:**
- **CSS Issues:**
- [ ] Post type should default to first one option

**Organize data:**
- **CSS Issues:**
- [ ] Column names not capitalized
- [ ] Dropdown width is too great

**Configure post:**
- **CSS Issues:**
- [ ] Not implemented yet

**Users:**
- **CSS Issues:**
- [ ] Search filters is shown on pattern library rather than Role filtering
- [ ] Filter is always shown
- [ ] View does not resize properly

**Comment by rjmackay** [comment]
Thanks @willdoran I'll break this down into smaller tasks (especially for the 'not implemented' things)

**Comment by rjmackay** [comment]
Closing in favour of continued mustang work
